zipline/utils/events.py

```diff
@@ -54,6 +58,9 @@
 MAX_WEEK_RANGE = 5
 
 
+_static_nyse_cal = get_calendar('NYSE')
```

Comment:
does this mean that none of the schedule function rules work if you use a different calendar?